recipes/user.rb

```diff
@@ -32,13 +32,16 @@
     gid user['group']
     home user['home']
     shell '/bin/bash'
-    manage_home true
+    manage_home user['home']['manage?']
   end
 
-  directory user['home'] + '/bin' do
-    owner user['name']
-    group user['group']
-    mode 0700
+  user['home']['directories'].each do |dir|
```

Comment:
I'm not interested in merging this.  Write a wrapper recipe and manage extra directories based on attributes.